Week1/homework/app.js

```diff
@@ -1,11 +1,206 @@
+        const languageText = infoOfBooks[key].language;
+        const authorText = infoOfBooks[key].author;
+        const titleElement = document.createElement('h1');
+        const languageElement = document.createElement('p');
+        const authorElement = document.createElement('p');
+        const li = document.createElement('li');
+        titleElement.innerText = titleText;
+        languageElement.innerText = languageText;
+        authorElement.innerText = authorText;
+        li.appendChild(titleElement);
+        li.appendChild(languageElement);
+        li.appendChild(authorElement);
+        ul.appendChild(li);
+    }
+}
+
+
+function booksId() {
+    const books = document.getElementById('books');
+    const ul = document.createElement('ul');
+    books.appendChild(ul);
+    createUlElement(ul);
+}
+window.addEventListener('load', booksId);*/
+
+
+//1.7 Find and download book covers for each book and construct a new object which has as keys the book IDs again,
+// and as value the path to the image source
+
+const images = {
+    romeo_and_juliet: './img/romeo-juliet.jpg',
+    harry_potter_and_the_sorcerer_stone: './img/harry potter.jpg',
+    the_book_of_css3: './img/css3.jpg',
+    catching_fire: './img/catching_fire_book_cover.jpg',
+    eloquent_javaScript: './img/javascript.jpg',
+    hamlet: './img/hamlet.jpg',
+    pro_html5_programming: './img/html5.jpg',
+    a_christmas_carol: './img/A Christmas Carol.jpg',
+    divergent: './img/divergent.jpg',
+    the_handmaid_tale: './img/The Handmaid Tale.jpg',
+
+}
+
+//1.8 Loop over these entries (hint: Object.keys(objectName) gives you an array containing the keys). 
+//Then write a function which places an image at the corresponding li element
+
+
+function createUlElement(ul) {
+    console.log(Object.keys(infoOfBooks))
+    Object.keys(infoOfBooks).forEach((key) => {
+        const titleText = infoOfBooks[key].title;
+        console.log(infoOfBooks[key].title)
+        const authorText = infoOfBooks[key].author;
+        const languageText = infoOfBooks[key].language;
+
+
+        const titleElement = document.createElement('h1');
+        const imageElement = document.createElement('img');
+        imageElement.setAttribute('src', images[key]);
+        imageElement.setAttribute('alt', key);
+        imageElement.setAttribute('width', '70%');
+        const languageElement = document.createElement('p');
+        const authorElement = document.createElement('p');
+        const li = document.createElement('li');
+        li.setAttribute('id', key);
+        titleElement.innerText = titleText;
+        authorElement.innerHTML = `<strong>Author: </strong>  ${authorText}`;
+        languageElement.innerText = "(" + languageText + ")";
+
+        li.appendChild(titleElement);
+        li.appendChild(imageElement);
+        li.appendChild(authorElement);
+        li.appendChild(languageElement);
+
+        ul.appendChild(li);
+    })
+}
+
+function booksId() {
```

Comment:
try to come up with a better name for this function, to make clear what this function is doing